transform/main.py

```diff
@@ -0,0 +1,34 @@
+import matplotlib.pyplot as plt
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver/module does.